packages/rangelink-vscode-extension/src/config/__tests__/DirtyBufferSettingMigrator.test.ts

```
@@ -0,0 +1,146 @@
+import { createMockConfigReader, createMockVscodeAdapter } from '../../__tests__/helpers';
+import { SETTING_UNSAVED_FILE_ACTION } from '../../constants';
+import { LEGACY_WARN_ON_DIRTY_BUFFER } from '../../constants';
```

ESLint

[error] Duplicate import from '../../constants'. Merge these imports into a single statement.

(barrel-imports/no-duplicate-barrel-imports)